openbb_terminal/mutual_funds/mutual_fund_controller.py

```diff
@@ -137,23 +106,423 @@ def print_help(self):
+    #     )
+    #     parser.add_argument(
+    #         "-b",
+    #         "--by",
+    #         choices=self.search_by_choices,
+    #         default="name",
+    #         dest="by",
+    #         help="Field to search by",
+    #     )
+    #     parser.add_argument(
+    #         "--fund",
+    #         help="Fund string to search for",
+    #         dest="fund",
+    #         type=str,
+    #         nargs="+",
+    #         required="-h" not in other_args,
+    #     )
+    #     parser.add_argument(
+    #         "-s",
+    #         "--sortby",
+    #         dest="sortby",
+    #         choices=self.search_cols,
+    #         help="Column to sort by",
+    #         default="name",
+    #     )
+    #     parser.add_argument(
+    #         "-l",
+    #         "--limit",
+    #         help="Number of search results to show",
+    #         type=check_positive,
+    #         dest="limit",
+    #         default=10,
+    #     )
+    #     parser.add_argument(
+    #         "-r",
+    #         "--reverse",
+    #         action="store_true",
+    #         dest="reverse",
+    #         default=False,
+    #         help=(
+    #             "Data is sorted in descending order by default. "
+    #             "Reverse flag will sort it in an ascending way. "
+    #             "Only works when raw data is displayed."
+    #         ),
+    #     )
+    #     if other_args and "-" not in other_args[0][0]:
+    #         other_args.insert(0, "--fund")
+    #     ns_parser = self.parse_known_args_and_warn(parser, other_args)
+    #     if ns_parser:
+    #         search_string = " ".join(ns_parser.fund)
+    #         investpy_view.display_search(
+    #             by=ns_parser.by,
+    #             value=search_string,
+    #             country=self.country,
+    #             limit=ns_parser.limit,
+    #             sortby=ns_parser.sortby,
+    #             ascend=ns_parser.reverse,
+    #         )
+    #     return self.queue
+
+    @log_start_end(log=logger)
+    def call_overview(self, other_args: List[str]):
+        """Process country command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="overview",
+            description="Show overview of funds from selected country.",
+        )
+        parser.add_argument(
+            "-l",
+            "--limit",
+            help="Number of search results to show",
+            type=check_positive,
+            dest="limit",
+            default=10,
+        )
+        ns_parser = self.parse_known_args_and_warn(
+            parser, other_args, export_allowed=EXPORT_ONLY_RAW_DATA_ALLOWED
+        )
+        if ns_parser:
+            investpy_view.display_overview(
+                self.country, limit=ns_parser.limit, export=ns_parser.export
+            )
+        return self.queue
```

Comment:
This should also be taken out.  Or if it still works, it can be added to the choices.